examples/CardsExample.php

```diff
@@ -15,11 +15,10 @@
 
 require 'vendor/autoload.php';
 
-Xendit::setApiKey(
-    <<<'TAG'
-xnd_development_prHUBDfVuOQTxyWTQSNkpjn9OwX9ZSUjdqgF9GenZ6hwhUQkc3NZ9WVexdH
-TAG
-);
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . '/..');
+$dotenv->load();
+
+Xendit::setApiKey(getenv('SECRET_API_KEY'));
```

Comment:
Is it possible to extract the logic into a different function? So it will not create duplication.

Reply:
yes, it's possible. thanks for the advice! code fix: [here](https://github.com/xendit/xendit-php-clients/pull/47/commits/d3b502d9051ff16effc2a98ce42eda72c57b4ac9)